### PR TITLE
145 swagger docs with examples

### PR DESCRIPTION
**Pull request by Schereo**
Added example request-bodies for all routes.

### CODE REVIEW

File: api/src/routers/channel-info/index.ts [hunk 1]
```diff
@@ -47,7 +47,31 @@ export const channelInfoRouter = Router();
  *       in: query
  *       required: false
  *       schema: 
- *         $ref: "#/components/schemas/ChannelInfoSearchSchema"
+ *         type: object
```
Comment (dominic22) [inline api/src/routers/channel-info/index.ts:50]: Why did you remove the ChannelInfoSearchSchema schema here?

File: api/src/routers/channel-info/index.ts [hunk 2]
```diff
@@ -47,7 +47,31 @@ export const channelInfoRouter = Router();
...
+ *           index:
+ *             type: number
+ *         example:
+ *           author: null
+ *           'topic-type': example-channel-data
+ *           'topic-source': null
+ *           created: null
+ *           'latest-message': null
+ *           limit: null
+ *           index: null
```
Comment (dominic22) [inline api/src/routers/channel-info/index.ts:74]: if possible it might be better to use undefined instead of null here

File: api/src/routers/subscription/index.ts [hunk 3]
```diff
@@ -77,14 +77,22 @@ subscriptionRouter.get('/subscription/:channelAddress', apiKeyMiddleware, authMi
  *         application/json:
  *           schema: 
  *             $ref: "#/components/schemas/RequestSubscriptionBodySchema"
+ *           example:
+ *             seed: qoekxjyzswfibbdgtvcxiigreptpnppzijkdyyeskftrqycqauengfyjobteluiudiqaucnh
```
Comment (dominic22) [inline api/src/routers/subscription/index.ts:81]: please leave the seed empty in this case it will be generated by the api

File: api/src/routers/verification/index.ts [hunk 4]
```diff
@@ -274,6 +316,8 @@ verificationRouter.post('/check-credential', apiKeyMiddleware, validate({ body:
  *         application/json:
  *           schema: 
  *             $ref: "#/components/schemas/RevokeVerificationBodySchema"
+ *           example: 
+ *             signatureValue: example-key
```
Comment (dominic22) [inline api/src/routers/verification/index.ts:320]: please use this as example:
`"signatureValue": "8VZJmBRcjjsmzoLGcdap5MWBqgKVf27Gbvubatk2ibxU.1112Pe57T4J3dzcCs5V2X2ffM8Xex521NkscDDoa8cshh7sSM.596PBh54fsBnHj9P5Fgw3mSJrChFNHDifb2mf4oK6kMBL8PU52c5f5oMHCrV3CudZ9PoexjRfrjL6TjAM22vkyYv"`

File: api/src/routers/channel-info/index.ts (hunk 1 above)
Comment (Schereo) [reply api/src/routers/channel-info/index.ts:50]: In the query the keys are written with dashes, in the schema the keys are in camelCase so the keys in the query are not compatible with the ones in the schema

File: api/src/routers/channel-info/index.ts (hunk 2 above)
Comment (Schereo) [reply api/src/routers/channel-info/index.ts:74]: To my knowledge undefined is only used in js. This is comment interpreted as yaml (json) so null is the value used for empty fields.

**Review by dominic22 — APPROVED** [review]
